conda/core/prefix_data.py

```diff
@@ -437,7 +438,12 @@ def get_conda_anchor_files_and_records(site_packages_short_path, python_records)
     ).match
 
     for prefix_record in python_records:
-        anchor_paths = tuple(fpath for fpath in prefix_record.files if matcher(fpath))
+        anchor_paths = []
+        for fpath in prefix_record.files:
+            if on_win:
+                fpath = fpath.replace("\\", "/")
+            if matcher(fpath):
+                anchor_paths.append(fpath)
```

Comment:
We could fix this earlier in the PrefixRecord instantiation, but then we would pay for this conversion every time we load any record. However we only need this fix for `conda list`, which is the only one that uses `PrefixData` with `pip_interop_enabled=True`. This function in only called in that code path.